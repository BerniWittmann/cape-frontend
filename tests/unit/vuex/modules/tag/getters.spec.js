import tagGetters from '@/vuex/modules/tag/getters'

describe('Vuex', () => {
  describe('Modules', () => {
    describe('Tag', () => {
      describe('Getters', () => {
        it('returns all getters', () => {
          expect(tagGetters).toMatchSnapshot()
        })
      })
    })
  })
})
