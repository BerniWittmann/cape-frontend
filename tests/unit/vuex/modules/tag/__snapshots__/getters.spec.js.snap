// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vuex Modules Tag Getters returns all getters 1`] = `Object {}`;
